client/stylesheets/puzzle.scss

```
@@ -257,65 +255,47 @@ table.puzzle-list {
 }
 
 .chat-input-row {
-  flex: none;
-  display: flex;
-  flex-direction: row;
-  align-items: stretch;
-  justify-content: space-between;
-  border-top: $input-border-width solid $input-border-color;
-  border-bottom: $input-border-width solid $input-border-color;
-  overflow: hidden;
-  &:focus-within {
-    border-color: $input-focus-border-color;
-    box-shadow: $input-focus-box-shadow;
-  }
-
-  textarea {
-    flex: 1 1 auto;
-    border: none;
-    &:focus {
-        box-shadow: none;
-    }
-  }
-
-  button {
-    border-radius: 0;
-  }
+  padding: $comm-pane-padding;
+  //capitalize Max to prefer css max over scss max
+  padding-bottom: Max(env(safe-area-inset-bottom, 0px), $comm-pane-padding);
 }
 
 .chatter-section {
   flex: 0;
   background-color: #ebd0e3;
   font-size: 12px;
   line-height: 12px;
-  border-bottom: 1px solid #000;
+  padding: $comm-pane-padding;
 
   .av-actions {
     display: flex;
     flex-direction: row;
-    //margin-top: 6px;
+    margin-bottom: $comm-pane-padding;
+    gap: $comm-pane-padding;
 
     .btn {
       flex: 1;
       padding: 0 auto;
       display: flex;
       align-items: center;
       justify-content: center;
-      border-radius: 0;
     }
   }
 }
 
 .chatter-subsection {
+  margin-top: 4px;
+
   header {
-    margin: 4px 2px;
+    margin: 4px 0;
     cursor: pointer;
   }
 
   .people-list {
     display: flex;
     flex-wrap: wrap;
     justify-content: flex-end;
```

I was tempted to change this, so chatting people flow left to right, but I didn't want to step on toes. @zarvox Why did you choose right to left?
```suggestion
    justify-content: flex-start;
```

---

I think that was @aldeka's design originally.  Karen, do you remember your reasons for having additional person-boxes fill space right-to-left?

I can appreciate that it feels like it makes better use of the horizontal space, and that in small calls the collapse controls are easier to read when not cramped by the people-boxes, but I haven't tried the opposite yet

---

Yeah, Karen confirms it's primarily for visual balance -- when there are few viewers, the left-right balance is much better, and when there are many viewers, it looks about the same either way.